src/components/common/FileUploadPreview.vue

```diff
@@ -77,6 +85,22 @@ export default class FileUploadPreview extends Vue {
   isFileValid () {
     return this.$refs.fileUploadInput.validate()
   }
+
+  @Watch('showErrors')
+  private onShowErrorsChanged (): void {
+    if (this.showErrors) {
+      this.$refs.fileUploadInput.validate()
+    }
+  }
+
+  @Watch('customErrorMessage')
+  private onCustomErrorMessage (val: string): void {
+    if (val) {
+      this.customErrorMessages = [val]
+    } else {
+      this.customErrorMessages = []
+    }
```

Comment:
No change required here and i think opinions on this varying from developer to developer 😆 
But this is a good example where a ternary might make sense! 
ie `this.customErrorMessages = val ? [val] : []`

Just less code but not everyone likes ternaries, so your call :)

Reply:
yes, much cleaner that way.  will update.

Reply:
updated to use `this.customErrorMessages = val ? [val] : []`

Reply:
LGTM